scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/test_hidden_deleted_pins.yaml

```diff
@@ -0,0 +1,150 @@
+# hidden and deleted pins are defined and described by IPC 7351B naming convention
+# http://ohm.bu.edu/~pbohn/__Engineering_Reference/pcb_layout/pcbmatrix/IPC-7x51%20&%20PCBM%20Land%20Pattern%20Naming%20Convention.pdf
+# hidden pins remove a pin location and pin number (for example, going from pad 1 to pad 3 with pin 2 missing from the package)
+# deleted pins remove a pin location but not pin number (for example, pin 2 is missing from the package but the footprint has pad 2)
+# deleted pins do not support custom numbering schemes https://github.com/pointhi/kicad-footprint-generator/pull/371
+
+FileHeader:
+  library_Suffix: 'TO_SOT_SMD'
+  device_type: 'SOT'
+
+# test add_quad_pad_border() in quad_dual_pad_border.py
+LQFP-48_7x7mm_P0.5mm:
```

Comment:
Please rename the test-cases so its clear what they are testing.
And make them minimal, `size_source` is not needed here.

Reply:
I recalled the `size_source` removal discussion from the BGA pad size PR. But I checked and the gullwing and SON generators fail the same way if this argument isn't given so I kept it the same (for now).

Reply:
Ah, I was not aware of that. Perhaps put something else there. The datasheet-link to a real part is confusion.

Reply:
I changed them to a tide. On the gullwing and SON generators they are assumed to be different than a manufacturer-specified footprint, and thus we need a datasheet or package drawing. The BGA generator could be used for a generic footprint or manufacturer-specific one. But either way, we still need some documentation against which to verify the footprint before we could add it to the library. So I don't see this as a problem in the general sense as some kind of documentation is required, but it is a little weird for test footprints.